Dockerfile

```
@@ -90,9 +91,19 @@ RUN cd /app/parsec && \
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 ENV PATH="/app/.cargo/bin:${PATH}"
 RUN rustup default 1.90.0
 RUN cargo install --locked kani-verifier && cargo kani setup
+
+# Install ParseRust
+RUN rustup install nightly-2024-10-01
+RUN rustup default nightly-2024-10-01
+RUN rustup component add llvm-tools rustc-dev
+
+COPY --chown=${USER_ID}:${GROUP_ID} parserust /app/parserust
+RUN cd /app/parserust && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)